nginx.conf

```diff
@@ -24,6 +24,21 @@ server {
         proxy_set_header X-Forwarded-Host $host;
     }
 
+    # Proxy /api/media
+    location /api/media {
+        rewrite ^/api/media/(.*) /$1 break;
+        proxy_set_header Host media-minio:9000;
```

Comment:
```suggestion
        proxy_set_header Host ${MINIO_ENDPOINT};
```

Reply:
or
```suggestion
        proxy_set_header Host $host;
```

Reply:
It took me a while to figure out that `$host` does not work in my case, will substitute that with the environment variable